composer.json

```
@@ -38,7 +38,7 @@
     },
     "require-dev": {
         "laminas/laminas-coding-standard": "~2.5.0",
-        "phpunit/phpunit": "^10.4",
+        "phpunit/phpunit": "^10.5.35 || 11.3",
```

```suggestion
        "phpunit/phpunit": "^10.5.35",
```

11 won't get installed here, due to other conflicts